.github/workflows/bundle-size.yml

```diff
@@ -4,6 +4,10 @@ on:
     pull_request:
     workflow_dispatch:
```

Comment:
Out of the scope of this PR but this trigger doesn't seem supported by `preactjs/compressed-size-action`. Should be removed

Reply:
Noted. Will try to address this separately.

Reply:
@fashuvo Let's try to create follow-up issues in order to not forget about these in the future.